app/controllers/items_controller.rb

```diff
@@ -27,6 +51,8 @@ def edit
   def create
     @item = Item.new(item_params)
 
+    puts @item
```

Comment:
is this diagnosis puts?

Reply:
yep, I'll remove it now that it's working